src/lkl/setup.c

```diff
@@ -681,14 +707,15 @@ static void lkl_mount_disk(
             sgxlkl_fail("Could not allocate memory for disk encryption key\n");
         for (size_t i = 0; i < disk->key_len; i++)
             // TODO verify that this is secure
+            // cwinter: it's not, but I think we don't need this at all!
```

Comment:
Can you elaborate?

Reply:
`rand()` is not a secure way to generate keys, especially if the seed isn't initialized to anything with entropy. If we need keys or randomness here, we should use the DRNG instructions. However, I don't think we need fresh keys here at all; @letmaik used it for tests at some point, but in future we'll have all keys taken care of before we get to this point in the code.

Reply:
You're right I guess, the key shouldn't be generated during "mounting" but somewhere earlier. Could you change the comment to say that this should move and that we need to use proper randomness?